kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerHttpException.java

```diff
@@ -233,8 +232,13 @@ private Map<String, Object> getErrorBodyAs(Retrofit retrofit) {
         retrofit.responseBodyConverter(Map.class, new Annotation[0]);
     try {
       return converter.convert(retrofit2Response.errorBody());
-    } catch (IOException e) {
-      throw new RuntimeException(e);
+    } catch (Exception e) {
+      log.debug(
+          "Exception occurred while parsing the HTTP error response body. Status code : {}, URL : {}, stack trace : ",
```

Comment:
Let's make this match the [retrofit1 error message](https://github.com/spinnaker/kork/blob/cf1bcaa69088c84ec3734f1ea952d199d11e91d3/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerHttpException.java#L95-L99)
```suggestion
          "unable to convert response to map ({}, {})",
```
although it's tempting to extend it to include the request method as well, like:
```
      log.debug(
          "unable to convert response to map ({} {}, {})",
          retrofit2Response.raw().request().method(),
          retrofit2Response.code(),
          retrofit2Response.raw().request().url(),
          e);
```

Reply:
There are more places where it's interesting to use the request method so perhaps let's leave that to a followup PR.